### DISCUSSION TITLE
水军

### DISCUSSION THREAD

**Discussion by OoTyleroO** [share121/inter-knot#1396] [其他]
![1723363165559](https://github.com/user-attachments/assets/c121c096-309d-4df2-9fcc-c73979b1a48f)
所谓「科学方法」是如此面目可憎。它将「事实」和「价值」拆分，并强令你接受所有「事实」——即使那是你最讨厌的「事实」！
在过去，人们会说「事实胜于一切」、「要面对事实」、「真理至上」…但好消息是，大众传媒引导了一场关乎「真相」的变革：从前这世上只有一种「真相」，但现在，我们可以仅凭喜好选择自己喜欢的「真相」。

「欢迎来到后真相时代！」                                                                                                                             -----《曜青体育》融媒体中心

**Comment by lllliufeng** [comment]
td

**Comment by jenemi** [comment]
[后真相时代.txt](https://github.com/user-attachments/files/16573095/default.txt)
#### 后真相时代   ——Hector MacDonald（著）

**Comment by dogofvita** [comment]
关注七圣召唤枫达杯喵，谢谢喵，你知道的，我一直是牢伯的粉丝，恭喜牢伯进入12强喵
